Gemfile

```diff
@@ -145,7 +145,7 @@ group :test do
   gem 'rspec-example_disabler', git: "https://github.com/finnlabs/rspec-example_disabler.git"
   gem 'capybara', '~> 2.3.0'
   gem 'capybara-screenshot'
-  gem 'selenium-webdriver', '~> 2.43.0'
+  gem 'selenium-webdriver'
```

Comment:
Because people tend to use "bundle update" instead of "bundle (install)", we tend to restrict the versions.